server/sonar-server/src/main/java/org/sonar/server/permission/PermissionService.java

```diff
@@ -76,6 +77,11 @@ public PermissionService(DbClient dbClient, PermissionRepository permissionRepos
     return GlobalPermissions.ALL;
   }
 
+  public UserWithPermissionQueryResult findUsersWithPermission(PermissionQuery query) {
```

Comment:
For PermissionDaoFacade, in fact permission needs a lot of refactoring as it's not testable in the current form.
That's why we need to reduce its complexity -> Remove usage of PermissionService is a baby step.